reactivex/operators/_map.py

```diff
@@ -65,7 +65,7 @@ def _identity(value: _T1, _: int) -> _T2:
 
     return compose(
         ops.zip_with_iterable(infinite()),
-        ops.starmap_indexed(_mapper_indexed),
+        ops.starmap(_mapper_indexed),
```

Comment:
This depended on the incorrect behavior of `starmap_indexed`. If it actually were to pass the index, it would pass it twice: once from `zip_with_iterable(infinite())` and once from `starmap_indexed`